src/App/Part.h

```diff
@@ -53,7 +53,7 @@ class AppExport Part : public App::GeoFeature, public App::OriginGroupExtension
     /// unique identifier of the Item
     App::PropertyUUID    Uid;
     /// material descriptions
-    App::PropertyMap     Material;
+    App::PropertyLink    Material;
```

Comment:
And where do you handle this change? What about existing documents that are loaded with a FC version that has this PR included?

Reply:
Hello,
I've transferred the topic to the forum as requested: https://forum.freecadweb.org/viewtopic.php?f=10&t=65379.
I've made some propositions for existing documents in the post, I'll modify my PR according to the feedback on the forum.

Reply:
I am not opposed to the change. My feedback is that when you change a property you must take care of existing documents.